client/gatsby-ssr.js

```diff
@@ -0,0 +1,25 @@
+
+/* this is a hack to fix `gatsby-plugin-sass` from including the
+  css in every single file (which is > 120k lines).
+  see: https://github.com/gatsbyjs/gatsby/issues/1526
+  `ssr` means server-side rendering.
+  */
+export const onPreRenderHTML = ({getHeadComponents}) => {
+  if (process.env.NODE_ENV !== 'production') {
+    return;
+  }
+
+  getHeadComponents().forEach((el) => {
+    // Remove inline css. https://github.com/gatsbyjs/gatsby/issues/1526
+    if (el.type === 'style') {
```

Comment:
This commenter needed to check on existence of `props['data-href']` - seems we might also want to do this?